backend/src/main/java/com/bakdata/conquery/io/csv/CsvIo.java

```diff
@@ -21,6 +22,10 @@ public static CsvWriter createWriter() {
 		return new CsvWriter(ConqueryConfig.getInstance().getCsv().createCsvWriterSettings());
 	}
 
+	public static CsvWriter createWriter(Writer writer) {
+		return new CsvWriter(writer, ConqueryConfig.getInstance().getCsv().createCsvWriterSettings());
+	}
+
 	public static boolean isGZipped(File file) throws IOException {
```

Comment:
in CSV.java existiert eine Methode mit dem gleichen Namen, aber anderer Logik Bitte die Implementationen vereinheitlichen